core/src/main/java/net/adoptopenjdk/icedteaweb/manifest/ManifestAttributesChecker.java

```diff
@@ -116,7 +116,11 @@ public void checkAll() throws LaunchException {
 
             if (attributesCheck.contains(MANIFEST_ATTRIBUTES_CHECK.PERMISSIONS) ||
                     attributesCheck.contains(MANIFEST_ATTRIBUTES_CHECK.ALL)) {
-                checkPermissionsAttribute();
+                if (file.getResources().getJARs().length > 0) {
```

Comment:
Could you please move this check into the method `checkPermissionsAttribute()`

Reply:
No problem, done.